tests/gmsh/create_tria_02.output.2

```diff
@@ -0,0 +1,105 @@
+
```

Comment:
Can you please guard both test outputs with: `.with_opencascade=true.output[.2]?`

Reply:
Something like
```
for (auto it : {iges_file_name, geo_file_name, msh_file_name, log_file_name, warnings_file_name})
  {
    // ...
  }
```

Reply:
You meant to attach that last comment here to the place where you say "Can we make this a loop"